### PR TITLE
Es6 promises | Adam Park

### PR DESCRIPTION
**Pull request by Adam-SungMin-Park**
Screen Shot 2021-01-13 at 12 25 34

### CODE REVIEW

File: es6-promises/index.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const takeAChance = require('./take-a-chance');
+
+const e = new Error('Another chance Adam');
+const p = takeAChance('Adam');
+
+p.then(resolve => {
+  console.log('Lucky day Adam!');
+}).catch(reject => {
+  console.log(e);
```
Comment (Zhiwutian) [inline es6-promises/index.js:9]: here you should be logging reject.message good sir!

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
tiny update!

File: es6-promises/index.js (hunk 1 above)
Comment (Adam-SungMin-Park) [reply es6-promises/index.js]: updated!

**Review by Zhiwutian — APPROVED** [review]
👍

**Pull request merged**